src/hugr/views.rs

```diff
@@ -333,14 +333,9 @@ pub trait HugrView: sealed::HugrInternals {
 
     /// For function-like HUGRs (DFG, FuncDefn, FuncDecl), report the function
     /// type. Otherwise return None.
-    fn get_function_type(&self) -> Option<PolyFuncType> {
+    fn get_function_type(&self) -> Option<FunctionType> {
```

Comment:
In the docstring - I prefer the wording that you've used in a couple of places, "signature of the inner dataflow (sibling) graph" (or "inner function type", but I think the wording with "signature" is good?)